catalogue/webapp/components/WorkDetails/ExplanatoryText.tsx

```diff
@@ -41,17 +51,25 @@ const ControlText = styled.span`
   text-decoration: underline;
 `;
 
-const Content = styled.div`
+type ContentProps = {
+  hidden: boolean;
+};
+
+const Content = styled.div<ContentProps>`
   display: ${props => (props.hidden ? 'none' : 'block')};
 `;
 
-type Props = {|
-  id: string,
-  controlText: string,
-  children: Element<any>,
-|};
+type Props = {
+  id: string;
+  controlText: string;
+  children: ReactElement;
```

Comment:
We can remove the explicit `children` field from props - `FunctionComponent` injects it for us

Reply:
We seem to leave it elsewhere, and in the interests of merging this rather than fighting TypeScript any further I'm going to merge as-is. We can sort this out everywhere later.